### PR TITLE
Update forkdaemon package to be php 7.2 compatible

### PR DESCRIPTION
**Pull request by samsouder**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "require": {
     "php": ">=5.3.0",
     "psr/log": "^1.0.0",
-    "barracudanetworks/forkdaemon-php": "^1.0.10"
+    "barracudanetworks/forkdaemon-php": "^1.1"
```
Comment (mrrsm) [inline composer.json]: This should be `^1.1.1` so we don't pull in non php 7.2 compliant version 1.1.0

**Review by mrrsm — DISMISSED** [review]

File: composer.json (hunk 1 above)
Comment (mrrsm) [reply composer.json]: I'm actually not sure if we want to change this. You technically have support for the later versions here.  We keep more support leaving it the way it is

**Comment by mrrsm** [comment]
I'm actually not sure if we want to change this. You technically have support for the later versions here. We keep more support leaving it the way it is

**Comment by samsouder** [comment]
You are right, the existing package version spec will match 1.1.1. Thanks.

**Pull request closed without merging**